taker-frontend/src/components/FundingRateTooltip.tsx

```diff
@@ -0,0 +1,23 @@
+import { Tooltip } from "@chakra-ui/react";
+import * as React from "react";
+
+interface FundingRateTooltipProps {
+    children: JSX.Element;
```

Comment:
You can achieve this with composition of an existing type: `PropsWithChildren`.

See https://www.mendelowski.com/docs/react/children-props-with-typescript/.

Reply:
cool, I didn't know about this. 👍

Reply:
works nicely. Need to remember this :)

Reply:
thanks for sharing this tip, looks really useful! 👍